### ISSUE TITLE
add more intuitive discussion of scaled

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/sourmash#2287]
asked on slack by Jonathan Rondeau-Leclaire -

>I am trying to understand the scaled argument in sourmash gather. Can you point me to what I need to read to understand it ? Or explain it to me quickly here ?

>I searched the github issues to help clarify and read [issue #1340](https://github.com/sourmash-bio/sourmash/issues/1340) [...] - but what about accuracy and algorithm performance?

The paper on it is here, https://www.biorxiv.org/content/10.1101/2022.01.11.475838v2 - it’s the s parameter in FracMinHash. The paper describes the loss of ability in accuracy when estimating overlap, containment, and Jaccard  similarity. But I accept that that is not the easiest paper to understand :joy: 

Perhaps the easiest way to think of it intuitively is that it’s like JPG or other lossy image compression techniques - you can slide the parameter from 1 (perfect image, no loss of information, but no real size reduction) to 100,000 (lots of information loss, but also lots of size reduction!).

There are really only two practical things to keep in mind with scaled -

first, a scaled of 1000 - 10,000 works perfectly well for detecting overlaps greater than 5000-50,000 bp between data sets. Since genomes tend to be much larger than 50,000 bp, you’re good with a scaled of 1000 or even 10,000, both of which result in a really substantial improvement in computational resources (literally a factor of x scaled reduction in memory, disk, and compute requirements).

second, if you’re using a prepared database like one of our genbank databases, you’re limited by to the larger size of the scaled used to prepare the database - you can’t download and use a database prepared at scaled=2000 with a genome sketch you built at a scaled of 100, but you CAN use a database prepared at a scaled of 2000 with a sketch built at a scaled of 10000.

tl;dr scaled = 1000 is perfectly good for bacterial and archaeal genomes, and even most viruses.
